### ISSUE TITLE
Why do you use custom stringify?

### ISSUE THREAD

**Issue by surik** [bitwalker/conform#59]
Why do you use custom stringify [here](https://github.com/bitwalker/conform/blob/master/lib/conform/utils/code.ex#L22) instead of `Macro.to_string`?

**Comment by bitwalker** [comment]
Because of formatting. `Macro.to_string` works fine of course for producing a string of source code, but it is very obviously formatted by the computer. I wanted to generate source code which closely matched how one might type it by hand, and in order to do that, I had to implement my own stringification.

**Comment by surik** [comment]
Ok, I see your point. 
But your custom stringification doesn't support all elixir syntax. For example '!=' in when :

```elixir
    "connect.uri": fn
      _, uri ->
        case :ex_uri.decode(uri) do
          {:error, :invalid_uri} when uri != 'local' ->
            exit("Unsupported URI format")
          _ ->
            uri
        end
```

It because [here](https://github.com/bitwalker/conform/blob/master/lib/conform/utils/code.ex#L149) is allowed only `in`.

**Comment by bitwalker** [comment]
Ah, indeed, I'll get that addressed ASAP.

**Comment by liveforeverx** [comment]
+1 for production usage `Macro.to_string/1` should be mostly always better-tested.

**Comment by bitwalker** [comment]
@liveforeverx Well that stringification is only used for generating a schema, it's not something used in production. In that case, it's better to have something that requires less fiddling by the user.

However it's been quite some time since I wrote that code, so perhaps there are ways to use stdlib functions to generate more idiomatic output these days, I'll take a look.

**Comment by surik** [comment]
Any results?

**Comment by bitwalker** [comment]
@surik The issue you brought up has been addressed in 2.1.1
